onebusaway-android/build.gradle

```diff
@@ -314,4 +321,7 @@ dependencies {
     googleCompile 'com.google.android.gms:play-services-maps:9.0.1'
     // Amazon Maps (only for Amazon flavor)
     amazonCompile 'com.amazon.android:amazon-maps-api:2.0'
+    // POJOs used for full data-binding via Jackson
```

Comment:
Can you please move this dependency above the `googleCompile`? line?  I've been trying to keep the dependencies used for all build variants above the ones used for only specific build variants.